src/js/Domain/ResultRacingDomain.js

```diff
@@ -0,0 +1,25 @@
+import RacingInfoDomain from "./RacingInfoDomain.js";
```

Comment:
```suggestion
import racingInfoDomain from "./RacingInfoDomain.js";
```
RacingInfoDomain은 class가 아니라 instance를 반환하고 있어요!
그래서 받아올때도 PascalCase가 아니라 camelCase로 표기해주면 어떨까요?